python/mlcroissant/pyproject.toml

```diff
@@ -35,12 +35,15 @@ readme = "README.md"
 # Development deps (linting, formatting,...)
 # Installed through `pip install -e .[dev]`
 dev = [
-  "black==23.11.0",
-  "datasets",
-  "flake8-docstrings",
+  "mlcroissant[devdeps]",
   "mlcroissant[git]",
   "mlcroissant[image]",
   "mlcroissant[parquet]",
+]
+devdeps = [
```

Comment:
Why do you need to distinguish between dev and devdeps and what distinction do you make?